device-types/Ruckus/r760.yaml

```diff
@@ -0,0 +1,31 @@
+---
+manufacturer: Ruckus
+model: R760
+slug: ruckus-r760
+part_number: 901-R760-XX00
+u_height: 0.0
+is_full_depth: false
+airflow: passive
+comments: '[Ruckus R750 Datasheet](https://www.commscope.com/product-type/enterprise-networking/wireless-access-points/indoor/r760/)'
+weight: 1.76
+weight_unit: kg
+console-ports:
+  - name: USB
+    type: usb-a
+power-ports:
+  - name: DC
+    type: dc-terminal
+    maximum_draw: 38
```

Comment:
Hello, thanks for the contribution. It appears that these PSU's are modular. These should be defined under `module-bays` rather than `power-ports`, and then the associated PSU modules should be created within the module-types folder if they do not already exist. Thank you.

Reply:
Hello, this is for a ruckus access point. Not  a server or switch. So PSU is all built in.
https://www.ruckusnetworks.com/products/wireless-access-points/r760/

Reply:
That's my bad, I misread this morning.